keps/sig-node/688-pod-overhead/README.md

```diff
@@ -311,12 +318,114 @@ Cons:
  * Not user perceptible from a workload perspective.
  * very complicated if the runtimeClass policy changes after workloads are running
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+Skipping this section as the feature was already rolled out to all supported k8s versions.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Using metrics mentioned in documentation https://kubernetes.io/docs/concepts/scheduling-eviction/pod-overhead/#observability:
+
+- `kube_pod_overhead_cpu_cores`
+- `kube_pod_overhead_memory_bytes`
+
+###### How can someone using this feature know that it is working for their instance?
+
+Using metrics mentioned in documentation https://kubernetes.io/docs/concepts/scheduling-eviction/pod-overhead/#observability:
+
+- `kube_pod_overhead_cpu_cores`
+- `kube_pod_overhead_memory_bytes`
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+N/A
```

Comment:
This isn't N/A, what does "normal quality of service" look like for this feature? How do we know it is performing as expected? Given the description of the proposal, I expect that this may cover the performance/expectations around updates of pod overhead and the mutating admission controller.

Reply:
updated